StripeIdentity/StripeIdentity/Source/API Bindings/Models/VerificationPage/VerificationPageStaticContentPhoneOtpPage.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  VerificationPageStaticContentPhoneOtpPage.swift
+//  StripeIdentity
+//
+//  Created by Chen Cen on 6/14/23.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+
+extension StripeAPI {
+
+    struct VerificationPageStaticContentPhoneOtpPage: Decodable, Equatable {
```

Comment:
if these come from the API, I'd double check the nullability (compare it vs. backend) because if something is marked as non-null when it could be null, there would be a decoding failure

Reply:
Just made [this](https://github.com/stripe/stripe-ios/compare/093440ede3d27c146687811431ad1626d0b51f5e..3f9f13b99cb843a2db640e037c5aed71bee44b23#diff-98c82b64f3dd675cafde9c2bf4791a5ba71983aa93be0e5e5d152455465e018c) change to guard the field to be non nil, and report an error otherwise